### DISCUSSION TITLE
Generate Problem

### DISCUSSION THREAD

**Discussion by giacomobandinelli** [nftchef/art-engine#68] [Help + Q&A] [answered]
Hello I have a problem, when I run "npm run generate" every image that I get is a mixture of all traits of all layers (I'll attach it). How do I solve this and get only 1 trait for each layer in every image? 
![Issue](https://user-images.githubusercontent.com/100479245/155847410-e71606c9-b98e-4a8d-923c-d21f4f3b0fcf.png)

**Comment by nftchef** [comment, marked as answer]
This is always one of my favorite errors, looks so fun!

Any image without a `#` weight will be required, resulting in a stack like this and running out of "DNA" options.
You will need to be sure to add a `#` to everything that should be "optional"
